01/app.js

```diff
@@ -31,8 +23,11 @@ const printData = function (inputArr) {
     usersList.appendChild(listItem);
 }
 
-const clearInput = function (currEl) {
-    currEl.value = '';
+const clearInput = function (currElList) {
+    for (let j = 0; j < currElList.length - 1; j++) {
+        console.log(currElList[j]);
```

Comment:
W takiej "ostateczniej" wersji, kiedy oddajesz projekt na rekrutacji to należy usunąć consologi :)